src/Pages/OfficerDB/Edits/Uploader.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+const api = require('../../../APIFunctions/OfficerDB')
+
+let file = ''
+
+export default function Uploader () {
```

Comment:
Id rather we not mix React components and utility functions in the same file, can you split this up? You can probably put the `Uploader` component back where it was.